packages/sdk-ark/source/ledger.service.ts

```diff
@@ -39,6 +41,12 @@ export class LedgerService extends Services.AbstractLedgerService {
 		return this.#transport.getVersion();
 	}
 
+	public override async isCompatible(): Promise<boolean> {
+		const currentVersion = await this.getVersion();
+
+		return compareVersion(currentVersion, this.#minimumVersion);
```

Comment:
```suggestion
		return compareVersion(await this.getVersion(), this.#minimumVersion);
```

Reply:
Calling `getVersion` probably requires the user to interact with the device?

Reply:
yeah exactly, we want to check the version of the app after opening the app and before going to the next page (import wallet)

basically after this line https://github.com/PayvoHQ/wallet/blob/develop/src/app/contexts/Ledger/hooks/connection.ts#L109